app/repository/download.py

```diff
@@ -5,39 +5,43 @@
 
 import pandas as pd
 from fastapi import Depends
+from sqlalchemy import bindparam, text
+from sqlalchemy.types import ARRAY, DATETIME, String
 
 from app.clients.db.session import get_db
 from app.repository.helpers import get_query_template
 
 _LOGGER = getLogger(__name__)
 
 
-def create_query(
-    template_query, ingest_cycle_start: str, allowed_corpora_ids: list[str]
-) -> str:
-    """Create download whole database query, replacing variables.
+def get_whole_database_dump(
+    ingest_cycle_start: str, allowed_corpora_ids: list[str], db=Depends(get_db)
+):
+    """Get whole database dump and bind variables.
 
     :param str ingest_cycle_start: The current ingest cycle date.
-    :param list[str] allowed_corpora_ids: The corpora from which we
+    :param list[str] corpora_ids: The corpora from which we
```

Comment:
The name in the docstring doesn't match the parameter name

Reply:
Ahh I merged before you finished your review by the looks of it as I didn't realise you were looking at it! I'm in the nav backend for my next ticket, so will update the docstring there - good spot!